requirements.txt

```diff
@@ -1,8 +1,8 @@
 click
 cython
-docarray>=0.4.4
+docarray>=0.13.16
+docarray[common]>=0.13.16
```

Comment:
Why having both ?

Reply:
To fix the issue `"google" module is not found`

Reply:
hmm okay !